ext/standard/var_unserializer.re

```diff
@@ -122,7 +121,7 @@ PHPAPI void var_push_dtor(php_unserialize_data_t *var_hashx, zval *rval)
 	}
 }
 
-PHPAPI zval *var_tmp_var(php_unserialize_data_t *var_hashx)
+static zend_always_inline zval *tmp_var(php_unserialize_data_t *var_hashx, int extra)
```

Comment:
From an API perspective I'd make this a `num` parameter, which is either 1 or 2, rather than the difference to 1.

Additionally it should also reserve num slots, i.e. also increment used_slots by the number. Currently it's an odd hybrid in that it only checks the allocation, but does not reserve the slots.

Reply:
Ah, much cleaner. Thanks!